FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -633,8 +632,7 @@ def export_getComponentDocumentation( self, cType, system, module ):
     except Exception as _e:
       return S_ERROR( 'No documentation was found' )
 
-  types_storeHostInfo = []
-  def export_storeHostInfo( self ):
+  def __storeHostInfo( self ):
```

Comment:
Yes, because the message is coming from the Framework, and it is not using the new S_ERROR structure.